client/app/dbaas/logs/detail/streams/alerts/streams-alerts.service.js

```diff
@@ -0,0 +1,183 @@
+class StreamsAlertsService {
+    constructor ($q, CloudPoll, OvhApiDbaas, ServiceHelper, StreamsAlertsConstant) {
+        this.$q = $q;
+        this.CloudPoll = CloudPoll;
+        this.OperationApiService = OvhApiDbaas.Logs().Operation().Lexi();
+        this.AlertsApiService = OvhApiDbaas.Logs().Alert().Lexi();
+        this.ServiceHelper = ServiceHelper;
+        this.StreamsAlertsConstant = StreamsAlertsConstant;
+    }
+
+    /**
+     * Adds a new alert
+     *
+     * @param {any} serviceName
+     * @param {any} streamId
+     * @param {any} alert - the alert object
+     * @returns promise which will be resolve to an operation object
+     * @memberof StreamsAlertsService
+     */
+    addAlert (serviceName, streamId, alert) {
+        return this.AlertsApiService.post({ serviceName, streamId }, alert).$promise
+            .then(operation => {
+                this._resetAllCache();
```

Comment:
There should be an interceptor in AlertsApiService that resets all cache when you POST something ^^

Reply:
Yes Mathieu. Its does have a interceptor and I have removed the redundant cache reset.